src/components/Sidebar/SectionSelection.test.js

```diff
@@ -45,4 +53,10 @@ describe('SectionSelection', () => {
 
     expect(addSectionMock).toHaveBeenCalledWith(defaultProps.sections[0]);
   });
+
+  it('disables sections that are already scheduled', () => {
+    const wrapper = getComponent({ scheduledSections: scheduledSectionsMock });
+
+    expect(wrapper).toMatchSnapshot();
```

Comment:
This is a good opportunity to think about robust unit testing: sure, this test works per say - but since we're already doing a render snapshot check in the first test of this file, doing another snapshot test isn't really needed here when we're trying to check the presence of a single prop. Cause in the end, that's all we really care about in this specific unit test, right? The fact that given some props, our component will render a `ListItem` with a disabled prop? So if we write a test to specifically test for that, then other changes to the component won't break this unit test, making it more robust.

You can use the [various shallow rendering methods from enzyme](https://airbnb.io/enzyme/docs/api/shallow.html) to achieve this. In this case, that would look something like:

```javascript
const wrapper = ...(what you have now)
expect(
  wrapper
    .findWhere(
      node => node.is(ListItem) && node.key() === scheduledSectionsTestData[0].id
    )
    .prop('disabled')
).toBe(true);
```

(Not 100% certain on the syntax here but the general idea holds - we want to use enzyme to find the specific `ListItem` component that we care about and assert that the disabled prop for that component should be set to true)

Also, let's not call the test data `scheduledSectionsMock`, since "mock" implies a mock function. `testScheduledSections` or `scheduledSectionsTestData` would be good I think.